src/azure-cli/setup.py

```diff
@@ -139,6 +139,7 @@
     'jsondiff==1.2.0',
     'packaging~=20.9',
     'PyGithub==1.38',
+    'PyNaCl==1.4.0',
```

Comment:
please also add in requirementxxxx files.

Reply:
It's already added in requirementsxxx files, so I just added here as well

Reply:
Why do we need to pin `PyNaCL`?

Reply:
unpinned